lib/fxpay/utils.js

```diff
@@ -39,27 +39,37 @@
     return a.origin || (a.protocol + '//' + a.host);
   };
 
+  exports.getCenteredCoordinates = function(w, h) {
+    // Centering calcs that work on multiple monitors (bug 1122683).
+    var x = window.screenX +
+      Math.max(0, Math.floor((window.innerWidth - w) / 2));
+    var y = window.screenY +
+      Math.max(0, Math.floor((window.innerHeight - h) / 2));
+    return [x, y];
+  };
+
+  exports.reCenterWindow = function(winRef, w, h) {
```

Comment:
it seems weird to me that this arg takes width and height but all other functions in this module use ``settings.winWidth`` directly. One thing you could do is allow optional width/height but fall back to the settings values.